content/docs/user-guide/networking/multiplayer/overview.md

```diff
@@ -45,14 +45,58 @@ Then, clients update themselves from a delta by:
+Developers can use *remote procedure calls* in O3DE to invoke a function on a remote endpoint. An RPC is a useful mechanism for signaling and notifying about events across networked endpoints. Unlike network properties, a developer chooses when to invoke an RPC. RPCs are not guaranteed to arrive in the order they are sent. O3DE offers both reliable and unreliable RPCs. By default, RPCs are reliable. 
 
 **Reliable** RPCs use a queue to guarantee the delivery of a message. When sending any reliable packet, the packet is also inserted into a priority queue and given a timeout value that's related to the latency of the connection that the message is sent on. On timeout, if the reliable packet was not explicitly acknowledged, the packet will be retransmitted. On the receiving end, the O3DE client tracks every reliable packet received and guarantees that any packet will only be delivered at most once. While this feature provides *guaranteed* delivery, it doesn't provide *ordered* delivery.
 
 **Unreliable** RPCs are sent over a "fire and forget" method. The host sending the message has no way to ensure that the message was received.
 
+### Types of RPCs
+
+Similar to network properties, RPCs have two important fields: `InvokeFrom` and `HandleOn`. These fields describe which role the RPC is invoked from and which role it's handled on.
+
+| Field | Description | Values | 
+| - | - | - |
+| `InvokeFrom` | Tells the network which role invokes this RPC. | Server, Autonomous, Client | 
+| `HandleOn` | Tells the network which roles handles this RPC. If Authority, the RPC is handled by the server that has authority over that entity. If Autonomous, the RPC is handled only by the relevant player's local client. If Client, the RPC is handled on all clients. | Authority, Autonomous, Client | 
+
+The following are four types of RPCs with a valid "invoked from and handled on" relationship and their possible use cases:
+
+- **Authority-to-Autonomous**: For example, sending corrections about the game state to the user.
+
+- **Authority-to-Client**: Authority sends calls to all the clients. For example, sending information about particle effects. 
+
+- **Server-to-Authority**: This is required to communicate information between entities. For example, suppose in a multi-server setup, EntityA is owned by ServerA and EntityB is owned by ServerB. If EntityA communicates directly to EntityB, EntityA will be talking to a proxy, not the real EntityB. Server-to-Authority ensures that messages always find the entity with authority. When a player wants to deal damage, Autonomous informs the Server, and the Server sends the DealDamage function to Authority. 
+
+- **Autonomous-to-Authority**: Used for sending user settings information that affects user input and is used during input-process time rather than input-creation time, such as mouse sensitivity and input controls. 
```

Comment:
Suggested text:
Used for sending user settings information...
to
Sends user settings information...